tests/uv/mlflow/test_reporter.py

```diff
@@ -36,7 +36,7 @@ def test_report_params():
     mlflow_cfg = {
         'experiment_name': 'foo',
         'run_name': 'bar',
-        'artifact_location': 'gs://foo/bar',
+        'artifact_location': '/foo/bar',
```

Comment:
changed the artifact locations here in the tests to avoid having to mock-out `google.auth.default`, and we test the artifact setup in the tests for `start_run`